lib/graphql/transaction.ts

```diff
@@ -5,50 +5,58 @@ import {
   TransactionsConnectionEdge
 } from "./schema";
 import { Model } from "./model";
+import { FetchOptions } from "./types";
+import { ResultPage } from "./resultPage";
+
+const safeGet = (source: any, path: any, defaultValue: any) =>
+  get(source, path, defaultValue) || defaultValue;
```

Comment:
Just out of curiosity: When do we need that?

Reply:
`get` returns `defaultValue` only when property from patch is undefined. Does not work like this in other falsy cases